test/ERC721A.test.js

```diff
@@ -5,319 +5,354 @@ const { ZERO_ADDRESS } = constants;
+              .withArgs(ZERO_ADDRESS, this.receiver.address, this.startTokenId);
+            await expect(mintTx).to.not.emit(this.receiver, 'Received');
+            expect(await this.erc721a.ownerOf(this.startTokenId)).to.equal(this.receiver.address);
+          });
 
-      it('successfully mints multiple tokens', async function () {
-        const mintTx = await this.erc721a.mint(this.receiver.address, 5, data, false);
-        for (let tokenId = 0; tokenId < 5; tokenId++) {
-          await expect(mintTx).to.emit(this.erc721a, 'Transfer').withArgs(ZERO_ADDRESS, this.receiver.address, tokenId);
-          await expect(mintTx).to.not.emit(this.receiver, 'Received');
-          expect(await this.erc721a.ownerOf(tokenId)).to.equal(this.receiver.address);
-        }
-      });
+          it('successfully mints multiple tokens', async function () {
+            const mintTx = await this.erc721a.mint(this.receiver.address, 5, data, false);
+            for (let tokenId = this.startTokenId; tokenId < 5 + this.startTokenId; tokenId++) {
+              await expect(mintTx)
+                .to.emit(this.erc721a, 'Transfer')
+                .withArgs(ZERO_ADDRESS, this.receiver.address, tokenId);
+              await expect(mintTx).to.not.emit(this.receiver, 'Received');
+              expect(await this.erc721a.ownerOf(tokenId)).to.equal(this.receiver.address);
+            }
+          });
 
-      it('does not revert for non-receivers', async function () {
-        const nonReceiver = this.erc721a;
-        await this.erc721a.mint(nonReceiver.address, 1, data, false);
-        expect(await this.erc721a.ownerOf(0)).to.equal(nonReceiver.address);
-      });
+          it('does not revert for non-receivers', async function () {
+            const nonReceiver = this.erc721a;
+            await this.erc721a.mint(nonReceiver.address, 1, data, false);
+            expect(await this.erc721a.ownerOf(this.startTokenId)).to.equal(nonReceiver.address);
+          });
 
-      it('rejects mints to the zero address', async function () {
-        await expect(this.erc721a.mint(ZERO_ADDRESS, 1, data, false)).to.be.revertedWith('MintToZeroAddress');
-      });
+          it('rejects mints to the zero address', async function () {
+            await expect(this.erc721a.mint(ZERO_ADDRESS, 1, data, false)).to.be.revertedWith('MintToZeroAddress');
+          });
 
-      it('requires quantity to be greater than 0', async function () {
-        await expect(this.erc721a.mint(this.owner.address, 0, data, false)).to.be.revertedWith('MintZeroQuantity');
+          it('requires quantity to be greater than 0', async function () {
+            await expect(this.erc721a.mint(this.owner.address, 0, data, false)).to.be.revertedWith('MintZeroQuantity');
+          });
+        });
       });
     });
-  });
-});
+  };
+describe('ERC721A', createTestSuite({ contract: 'ERC721AMock', constructorArgs: ['Azuki', 'AZUKI'] }));
+describe(
```

Comment:
Styling nit: can you add a new line between lines 354 and 355?

Reply:
Sure done ✅  Good catch 👍